lbz/_cfg.py

```diff
@@ -0,0 +1,21 @@
+import json
+
+from lbz.configuration import EnvValue
+
+# LBZ configuration
+LBZ_DEBUG_MODE = EnvValue("LBZ_DEBUG_MODE", default=False, parser=lambda a: a == "true")
+LOGGING_LEVEL = EnvValue("LOGGING_LEVEL", default="INFO")
+CORS_HEADERS = EnvValue("CORS_HEADERS", default=[], parser=lambda a: a.split(","))
+CORS_ORIGIN = EnvValue("CORS_ORIGIN", default=[], parser=lambda a: a.split(","))
+
+# AWS related configuration
+AWS_LAMBDA_FUNCTION_NAME = EnvValue("AWS_LAMBDA_FUNCTION_NAME", default="lbz-event-api").value
```

Comment:
Once this variable became more generic, I don't think we should put the default value to `lbz-event-api`.

Maybe, we shouldn't use the default value at all - this would make it required to be set if this part of the code is used 🤔